mods/gen/rules/vehicles.yaml

```diff
@@ -224,11 +224,6 @@ vehicle.humvee:
 		Weapons: debris.humvee.turret
 		Pieces: 1, 1
 		Range: 1c0, 1c0
-	ThrowsShrapnel@Turret2:
-		Weapons: debris.rocket_buggy.turret
-		Pieces: 1, 1
-		Range: 1c0, 1c0
-		RequiresCondition: tow_missile
```

Comment:
I can't check the new artwork rn, but it should still throw the Turret on death imo. After all even the small MG Turret is throw rn. Just needs a new weapon.

Reply:
Just add a white line on the turret, so throwing that will be weird